src/commands/update_forging_status.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import transactions from '../utils/transactions';
```

Comment:
OK for this PR, but why don't we just remove this file?

Reply:
Yes i think we should remove this file, but I don't think we need to refactor at this point